chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -57,29 +80,15 @@ Zotero.QuickCopy = new function() {
 		}
 		
 		return '';
-	}
-	
-	
-	/*
-	 * Returns the setting with any contentType stripped from the mode part
-	 */
-	function getContentType(setting) {
-		var matches = setting.match(/(?:bibliography|export)\/([^=]+)=.+$/, '$1');
-		return matches ? matches[1] : '';
-	}
-	
-	
-	/*
-	 * Returns the setting with any contentType stripped from the mode part
-	 */
-	function stripContentType(setting) {
-		return setting.replace(/(bibliography|export)(?:\/[^=]+)?=(.+)$/, '$1=$2');
-	}
+	};
 	
 	
-	function getFormatFromURL(url) {
+	this.getFormatFromURL = function(url) {
+		var quickCopyPref = Zotero.Prefs.get("export.quickCopy.setting");
+		quickCopyPref = JSON.stringify(this.unserializeSetting(quickCopyPref));
```

Comment:
@dstillman, maybe we should just do the conversion once on Zotero startup instead of scattering this code everywhere.